src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/Commands/Analyze/ConsoleAnalyze.cs

```diff
@@ -18,17 +19,19 @@ public class ConsoleAnalyze : IAppCommand
         private readonly IEnumerable<IAnalyzeResultProvider> _providers;
         private readonly IAnalyzeResultWriter _writer;
         private readonly IExtensionProvider _extensionProvider;
+        private readonly IOptions<AnalysisOptions> _options;
 
         public ConsoleAnalyze(
             IEnumerable<IAnalyzeResultProvider> analysisProviders,
             IUpgradeContextFactory contextFactory,
             IUpgradeStateManager stateManager,
             IAnalyzeResultWriter writer,
-            IExtensionProvider extensionProvider)
+            IExtensionProvider extensionProvider, IOptions<AnalysisOptions> options)
```

Comment:
nit: follow the pattern of the others (either all on the same line, or on an additional)